etc/test-helper.js

```diff
@@ -46,16 +46,16 @@ function verifyDocumentAccepted(doc, oldDoc, expectedChannels) {
   }
 }
 
-function verifyDocumentCreated(doc, oldDoc) {
-  verifyDocumentAccepted(doc, oldDoc, 'add');
+function verifyDocumentCreated(doc, expectedChannels) {
+  verifyDocumentAccepted(doc, undefined, expectedChannels || 'add');
 }
 
-function verifyDocumentReplaced(doc, oldDoc) {
-  verifyDocumentAccepted(doc, oldDoc, 'replace');
+function verifyDocumentReplaced(doc, oldDoc, expectedChannels) {
+  verifyDocumentAccepted(doc, oldDoc, expectedChannels || 'replace');
 }
 
-function verifyDocumentDeleted(doc, oldDoc) {
-  verifyDocumentAccepted(doc, oldDoc, 'remove');
+function verifyDocumentDeleted(oldDoc, expectedChannels) {
```

Comment:
There's no longer a need for the caller to include the updated document when deleting an existing document.